slemicro/combustion/script

```diff
@@ -232,7 +232,11 @@ if [ "${RANCHER}" = true ]; then
 	while ! kubectl wait --for condition=ready node $(cat /etc/hostname | tr '[:upper:]' '[:lower:]') --timeout=60s; do sleep 2 ; done
 
 	# https://ranchermanager.docs.rancher.com/pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster
-	helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+	if [ "${RANCHERPRIME}" = true ]; then
```

Comment:
```shell
if [ "${RANCHER}" = true ]; then
``` 
should be ${RANCHER} OR ${RANCHERPRIME} to get into this section

Reply:
Nope because the `RANCHER` flag needs to be set as well. The idea is to install rancher upstream by default (aka RANCHERPRIME=false) but giving the user the ability to install prime (BOTH RANCHER and RANCHERPRIME vars)